### PR TITLE
[add] py-ansimarkup: new recipe required by py-cylc-flow

### PR DESCRIPTION
**Pull request by LydDeb**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding @adamjstewart

File: var/spack/repos/builtin/packages/py-ansimarkup/package.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    """Produce colored terminal text with an xml-like markup."""
+
+    homepage = "https://github.com/gvalkov/python-ansimarkup"
+    pypi = "ansimarkup/ansimarkup-1.5.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("1.5.0", sha256="96c65d75bbed07d3dcbda8dbede8c2252c984f90d0ca07434b88a6bbf345fad3")
+
+    depends_on("python@2.7,3.3:3.9", type=("build", "run"))
```
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-ansimarkup/package.py]: Where do you get the upper restriction from?

**Review by manuelakuhn — COMMENTED** [review]
`py-cylc-flow` requires `py-ansimarkup@1:` so instead of version 1.5.0 you can also add a newer/the newest version.

File: var/spack/repos/builtin/packages/py-ansimarkup/package.py (hunk 1 above)
Comment (LydDeb) [reply var/spack/repos/builtin/packages/py-ansimarkup/package.py]: Thank you for your review. I add the latest version of ansimarkup and remove the dependency on python (I get the upper restriction from the setup.py but @adamjstewart pointed out to me not to take into account the content of classifiers.)

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**